docs/understand/driver_api.rst

```diff
@@ -0,0 +1,21 @@
+.. meta::
+  :description: This chapter will present the conception of driver API.
+  :keywords: AMD, ROCm, HIP, CUDA, driver API
+
+.. _driver_api:
+
+*******************************************************************************
+Driver API 
+*******************************************************************************
+
+The driver API offers developers low-level control over GPU operations, enabling them to manage GPU resources, load and launch kernels, and handle memory explicitly. This API is more flexible and powerful compared to the runtime API, but it requires a deeper understanding of the GPU architecture and more detailed management.
```

Comment:
It's important here to make the distinction between CUDA and HIP - CUDA has a separate driver API, whereas HIP includes all of this in its runtime API.

This goes for the whole file. A driver API in that sense does not exist for HIP.

Reply:
Moved to Programming interface PR. Fixing the problem there.

https://github.com/ROCm/HIP/pull/3532